sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -10548,4 +10548,17 @@ ALTER TABLE travelentry ALTER COLUMN dateofarrival SET NOT NULL;
 
 INSERT INTO schema_version (version_number, comment) VALUES (449, 'Add dateOfArrival to travel entries #7845');
 
+-- 2022-03-14 - #7774 Automatic & manual archiving for all core entities
+UPDATE contact SET archived = TRUE
```

Comment:
Please double check these update scripts, especially concerning contacts that don't have a source case; looks like for those, archived is also set to TRUE which definitely shouldn't happen.

Reply:
changed